src/rp2_common/hardware_gpio/include/hardware/gpio.h

```diff
@@ -1403,12 +1403,12 @@ extern void gpio_debug_pins_init(void);
 #endif
 
 
-// PICO_CONFIG: PICO_DEBUG_PIN_BASE, First pin to use for debug output (if enabled), min=0, max=28, default=19, group=hardware_gpio
+// PICO_CONFIG: PICO_DEBUG_PIN_BASE, First pin to use for debug output (if enabled), min=0, max=NUM_BANK0_GPIOS - 1, default=19, group=hardware_gpio
```

Comment:
this is actually 0-31 still

Reply:
Ah, thanks!

Reply:
Actually, I'm not able to test it, but would what I've done in the latest commit actually work? :thinking: